src/kuzzle.js

```diff
@@ -57,7 +57,8 @@ module.exports = Kuzzle = function (host, options, cb) {
         jwtTokenExpired: {lastEmitted: null, listeners: []},
         loginAttempt: {lastEmitted: null, listeners: []},
         offlineQueuePush: {listeners: []},
-        offlineQueuePop: {listeners: []}
+        offlineQueuePop: {listeners: []},
+        queryError: {lastEmitted: null, listeners: []}
```

Comment:
Adding the `lastEmitted` property automatically prevents this event being fired more than once every `this.eventTimeout` ms (currently set to 200ms).

We may not want to throttle this.